### PR TITLE
dependency and test update

### PR DESCRIPTION
**Pull request by eeswr**
tests were timing out on seneca 2.0, all passing on 2 and 3 now.

### CODE REVIEW

**Comment by mihaidma** [comment]
some updates needed please:
- please rebase,
- please update the deps to match the seneca ones

File: package.json [hunk 1]
```diff
@@ -23,15 +26,15 @@
...
   },
   "devDependencies": {
-    "eslint-config-seneca": "1.1.2",
-    "eslint-plugin-standard": "1.3.1",
+    "eslint-config-seneca": "1.x.x",
+    "eslint-plugin-standard": "1.x.x",
     "lab": "6.2.x",
-    "seneca": "senecajs/seneca",
-    "seneca-entity": "senecajs/seneca-entity"
+    "seneca": "^2.0.0",
```
Comment (shanel262) [inline package.json]: This needs to be 'plugin' and deps need updating.

File: package.json [hunk 2]
```diff
@@ -23,15 +26,15 @@
...
+    "seneca-entity": "1.2.0"
   },
   "peerDependencies": {
     "seneca": "*"
```
Comment (mihaidma) [inline package.json:40]: remove peer deps

**Comment by mihaidma** [comment]
will close this, deprecated

**Pull request closed without merging**